Makefile.toml

```diff
@@ -0,0 +1,79 @@
+[config]
```

Comment:
Fantastic @Dhole! :)

Reply:
@adria0 I wasn't considering adding this to this PR, but it just got included by an unconscious `git add .`.  This is a configuration file for [`cargo make`](https://github.com/sagiegurari/cargo-make/) (installed via `cargo install --force cargo-make`).  It allows defining a set of tasks to be run with the `cargo make` command easily.  For now I've added the build tasks, test tasks and code coverage using a different tool than the one defined in the github actions (`cargo tarpaulin` instead of `grcov`).  This allows running the tasks defined in github actions easily locally, just do `cargo make`.